### PR TITLE
Add few words about collecting and processing GH manifest files

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -441,6 +441,47 @@ To get info about the current session, access `/api/v1/authorized` endpoint.
...
+curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' --header 'auth-token: <your-token-here>' -d '{ \
+   "repositories": [ \
+     { \
+       "ecosystem": "maven", \
+       "force": true, \
+       "force_graph_sync": false, \
+       "recursive_limit": 0, \
+       "repo_name": "omalley/base64" \
+     } \
+   ] \
```
Comment (fridex) [inline README.md]: Backslashes shouldn't be there (they are wrongly produced by swagger) - here and bellow as well.

**Comment by msrb** [comment]
Related to https://github.com/openshiftio/openshift.io/issues/772.

File: README.md (hunk 1 above)
Comment (msrb) [reply README.md]: Good catch, let me fix that 😉

Comment (msrb) [reply README.md]: Backslashes removed.

**Pull request merged**